README.md

```diff
@@ -256,8 +256,7 @@ semantics onto C++ such as Rust-inspired
 
 ## Getting started
 
-You can get started playing with Carbon by checking out the codebase and using
-the Carbon explorer:
+You can get started playing with Carbon by [opening the project](https://gitpod.io#https://github.com/carbon-language/carbon-lang) in Gitpod via checking out the codebase and using the Carbon explorer:
```

Comment:
I will let the maintainers chime in on whether a GitPod setup is wanted or not, but I suggest that a GitPod link should be written down as an alternative to traditional cloning instead of replacing the old text.

Maybe something along the lines of "You can also open the Carbon project in GitPod <link>"

Reply:
👍🤔 💭

About to push the following

> If you do not wish to install software on your computer to try out or contribute to Carbon then you can also [open the project in Gitpod](https://gitpod.io#https://github.com/carbon-language/carbon-lang) which automates all of the steps above.

The language above creates a pathway for including GitHub Codespaces (nb:  they do not have a free/open source tier and [costs money immediately](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces)) if someone else wishes to append to the end as a list of options.